src/screens/home/components/HomeView.js

```diff
@@ -89,6 +89,15 @@ const HomeView = ({ route, navigation, setLauncherContext, setBarStyle }) => {
           const url = new URL(uri)
           url.hash = `/connected/${konnectorParam}`
 
+          const accountParam = consumeRouteParameter(
```

Comment:
Note for future me: I'm wondering if we can remove this `navigate(..., {konnector:XXX})` mechanic as we have also `mainAppFallbackURL` param that can do similar behavior (not sure)

cc @acezard